Fixed pathing for all the sidebar icons

erhuang623 opened the pull request:
## Pull Request

**Description**
Got rid of loading sidebar with JS when not needed.

**Type of change**
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

**Related Issues**
Please link the issue this pull request addresses. If applicable, add 'Fixes #<issue number>'.

**Screenshots**
If applicable, add screenshots to help explain your changes.

**Additional context**
Add any other context or screenshots about the pull request here.

erhuang623 committed:
Fixed pathing for all the sidebar icons

erhuang623 committed:
Fixed default and small colors

erhuang623 committed:
Merge branch 'main' into themeSwitcher

erhuang623 committed:
Changed highlight color and css variable

erhuang623 committed:
Fixing constant variable

ulises0516 reviewed (dismissed):
Great Catch! Now we dont have to rely on JS to load and make our website application have more immediate responses. Your work is fantastic!

erhuang623 committed:
Fixing waiting for the DOM to load

The code log wasn't waiting so it had a small flash of a default page. Should be fixed now

erhuang623 committed:
Fixed visual bug

The calendar dates displayed a rectangle when being deselected for a split second.

erhuang623 committed:
Fixed gradient on codelog sidebar

KaustubhPaliwal reviewed (approved):
Thanks for making this change. It was a good decision at the start of the quarter but now this should make the performance of the app much better and maintain consistency in the icons.

kevku reviewed (approved):
Nice! These changes will reduce the script loads and improve the speeds. This is following the RAIL model.
